.travis.yml

```diff
@@ -1,21 +1,22 @@
 language: php
 
+addons:
+    chrome: stable
+
 sudo: false
 
 php:
-    - "5.4"
-    - "5.5"
-    - "5.6"
-    - "7.0"
     - "7.1"
     - "7.2"
+    - "7.3"
+    - "7.4"
     - "nightly"
 
 env:
     matrix:
-        - SYMFONY_VERSION=2
```

Comment:
Does this patch break the compatibility with Sf2 applications?

Reply:
The support of sf2 is finish